src/components/Charts/ChartZones.tsx

```diff
@@ -77,7 +79,8 @@ const ChartZones = ({
   const maxDate = new Date();
   assert(minDate !== undefined, 'Data must not be empty');
 
-  const xScale = getZonesTimeScale(minDate, maxDate, 0, chartWidth);
+  const xScale = getUtcScale(minDate, maxDate, 0, chartWidth);
+  const dateTicks = getTimeAxisTicks(minDate, maxDate);
```

Comment:
We need to use the date range from from `getUtcScale`, because the endDate is of the scale is changed here [src/components/Charts/utils.tsL159](https://github.com/covid-projections/covid-projections/pull/2657/files#diff-94fef4efde3155913fc63fdabe97e9081a90d10b28fd2107de31826e0f239f5fR159)

```ts
const [startDate, endDate] = xScale.domain();
const dateTicks = getTimeAxisTicks(startDate, endDate);
```